book/testing.rst

```diff
@@ -17,8 +17,9 @@ it has its own excellent `documentation`_.
 
 .. note::
 
-    It's recommended to use the latest stable PHPUnit version (you will have
-    to use version 4.2 or higher to test the Symfony core code itself).
+    It's recommended to use the latest stable PHPUnit version, `installed as
+    PHAR`_ (you will have to use version 4.2 or higher to test the Symfony core 
+    code itself).
```

Comment:
Oh well, now that I look at this again: I think we should remove the note about the core as testing Symfony itself is a bit different (we have our own PHPUnit wrapper script).